packages/tabs-react/documentation/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html style="height: 100%">
+    <head>
+        <meta charset="utf-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <title>Jøkul Tabs Example</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+    </head>
+
+    <body style="height: 100%; display: flex; align-items: center; justify-content: center">
+        <div id="app" class="jkl"></div>
+        <script src="index.tsx"></script>
```

Comment:
Devserveren må endres litt etter oppdateringen til Parcel 2. Kanskje noe annet jeg glemmer @joms? Får den ikke til å kjøre på min localhost.

```suggestion
        <script type="module" src="index.tsx"></script>
```

Reply:
Måtte fjerne style attributter i HTML filen i tillegg.